cheatsheets/DotNet_Security_Cheat_Sheet.md

```diff
@@ -246,7 +248,226 @@ EXEC strQry // SQL Injection vulnerability!
 
 DO: Practise Least Privilege - Connect to the database using an account with a minimum set of permissions required to do it's job i.e. not the sa account
 
-## A2 Weak Account management
+### [OS Injection](OS_Command_Injection_Defense_Cheat_Sheet.md#net)
```

Comment:
Do not put link in section title, instead, add a line like this:

*Information about this issue can be found on this cheat sheet* and add a relative link to the dedicated cheat sheet.